### PR TITLE
vet_clinic database

### PR DESCRIPTION
**Pull request by rahimialijan**
# vet_clinic database (Animals Table)

- Insert new values for the animals' table
- Add a new column for the table
- Add update and delete transaction
- Do some queries for the animals' table
All Queries Results
rollback for delete fucn
savepoint
update_spacies column

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -4,4 +4,13 @@ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
...
        ('Gabumon', '2018-11-15', 2, true, 8),
        ('Pikachu', '2021-01-07', 1, false, 15.04),
-       ('Devimon', '2017-05-12', 5, true, 11);
+       ('Devimon', '2017-05-12', 5, true, 11),
+       ('Charmander', '2020-02-08', 0, false, 11),
+----------------------------insert new values for animals table----------------------------------------
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES ('Plantmon', '2021-11-15', 2, true, 5.7),
+        ('Squirtle', '1993-04-02', 3, false, 12.13),
+        ('Angemon', '2005-06-12', 1, true, 45),
```
Comment (bobb-Rob) [inline data.sql]: - [ ] You should insert the correct weight data for Charmander `-11`, Plantmon - `-5.7`, Squirtle - `-12.13`, and Angemon - `-45`. They all should be negative values.

**Review by bobb-Rob — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️
Hi @rahimialijan, 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you are almost there!  😃 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Highlights
✅ No Linter error
✅ Correct use of Gitflow
✅ Very descriptive Pull Request(PR)

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (rahimialijan) [reply data.sql]: thanks for your review, done

**Review by danmainah — APPROVED** [review]
Hi @rahimialijan , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ Added the data correctly 
✔️  Added all queries correctly, with screenshots of the query results

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**